src/cmd/launch.rs

```diff
@@ -22,3 +126,228 @@ impl LaunchStruct {
         LaunchStruct
     }
 }
+
+/// デバックコンテナを起動するために必要なディレクトリ群を初期化
+///
+/// - `~/.injesh/containers/<CONTAINER_NAME>/upper`
+///
+///     デバックコンテナ起動前の`/var/lib/docker/overlay2/<HASH_ID>/upper`を保存しておくためのディレクトリ
+///
+/// - `~/.injesh/containers/<CONTAINER_NAME>/setting.yaml`
+///
+///     デバックコンテナの`rootfs`やデバックコンテナ作成後に実行するコマンドなどを保存する設定ファイル
+fn initialize_setting<DO: Downloader>(
+    launch: &command::Launch<DO>,
+) -> Result<(), Box<dyn std::error::Error>> {
+    let user = user::User::new()?;
+    let dcontainer_base = format!("{}/{}", user.containers(), launch.name());
+
+    // rootfsの種類などが記載された設定ファイルsetting.yamlを~/.injesh/containers/に作成する
+    match Path::new(&dcontainer_base).exists() {
+        true => return Err(Error::AlreadyExists)?,
+        false => {
+            fs::create_dir_all(format!("{}/upper", &dcontainer_base))?;
```

Comment:
- boolで分岐するならmatchではなく if文を使う
- Errの早期リターンするなら `return` でなく `?` を使う
    - releted review: https://github.com/higuruchi/injesh/pull/43/files#r829899132

```suggestion
    if Path::new(&dcontainer_base).exists() {
        Err(Error::AlreadyExists)?
    } else {
        fs::create_dir_all(format!("{}/upper", &dcontainer_base))?;
```